### PR TITLE
many: misc tweaks

### PR DESCRIPTION
**Pull request by anonymouse64**
These are some small changes I split out from a larger WIP branch I have to hopefully make that one easier to review.

### CODE REVIEW

File: snap/info.go [hunk 1]
```diff
@@ -409,6 +409,9 @@ func (s *Info) Description() string {
 	return s.OriginalDescription
 }
 
+// GetType returns the type of the snap, specifically also returning TypeSnapd
+// if the snap is the snapd snap, regardless of the actual type of the snapd
```
Comment (bboozzoo) [inline snap/info.go]: >  regardless of the actual type of the snapd

Wondering about the wording here, doesn't fit exactly with the code.  Perhaps something like:

```
// GetType returns the type of the snap, including additional snap ID check
// for the legacy snapd snap definitions.
```
wdyt?

Comment (pedronis) [reply snap/info.go]: missing . at the end.  Maciej suggestion would also work for me.

File: snap/info.go [hunk 2]
```diff
@@ -787,6 +790,7 @@ func (st StopModeType) KillSignal() string {
 	return strings.ToUpper(strings.TrimSuffix(string(st), "-all"))
 }
 
+// Validate ensures that the StopModeType has an valid value
```
Comment (pedronis) [inline snap/info.go]: missing .

File: snap/info.go (hunk 1 above)
Comment (bboozzoo) [reply snap/info.go]: Pushed this along with a tweak for the other comment.

**Pull request merged**